### ISSUE TITLE
video_player_media_kit，VideoPlayer There is a problem with initialization completion

### ISSUE THREAD

**Issue by longgong** [media-kit/media-kit#599]
VideoPlayerController，After the asynchronous call of the initialize method is completed, the obtained width and height are 0. 
There is something wrong with the following code. After the initialization is completed, the value is always 0.

```
streamSubscriptions.add(
        player.stream.videoParams.listen(
          (event) {
            width = event.dw ?? 0;  //  adjusted to  //  event.dw；
            height = event.dh ?? 0; //  adjusted to  //  event.dh ；
            if ((width ?? 0) > 0 && (height ?? 0) > 0) {
              notify();
            }
          },
        ),
      );
      streamSubscriptions.add(
        player.stream.tracks.listen(
          (event) {
            // No video track is available i.e. an audio file.
            if (event.video.length == 2) {
              width = 0;
              height = 0;
              notify();
            }
          },
        ),
      );
```

**Comment by alexmercerind** [comment]
I can't really reproduce.

**Comment by alexmercerind** [comment]
Apologies for previous comment. I have raised a pull-request with the fix:

- #601

It used to work... I don't know what broke it now.

**Comment by alexmercerind** [comment]
@longgong 

We have published video_player_media_kit: ^1.0.5:
- https://pub.dev/packages/video_player_media_kit/versions/1.0.5

Let us know if your issue is resolved.

**Comment by longgong** [comment]
very good
